src/CMakeLists.txt

```diff
@@ -11,154 +11,6 @@ cmake_minimum_required(VERSION 3.1)
 
 include(CMakeParseArguments)
 
-## serialbox_add_library
-## ---------------------
-##
-## Create an object library with target "<TARGET>Objects" and convert them to a static and/or 
-## shared library with name "<NAME>.a" and <NAME>.so" respectively.
-## 
-##    NAME:STRING=<>             - Name of the library (e.g SerialboxCore)
-##    TARGET:STRING=<>           - Target used by CMake (e.g SerialboxCoreLibrary)
-##    OBJECTS:STRING=<>          - Object libraries to include (e.g SerialboxCoreLibraryObjects)
-##    NATIVE_PYTHON_LIB:BOOL=<>  - Generate a library which can be nativly imported in Python3. This
-##                                 means the library will have no prefix and the suffix is set to 
-##                                 .so
-##    BUILD_STATIC:BOOL=<>       - Convert object library to static library
-##    BUILD_SHARED:BOOL=<>       - Convert object library to shared library
-##    SYMLINKS:STRING=<>         - Add symlinks of the shared library to the specified paths
-##    LIBRARIES:=<>              - Specify external libraries or flags to be use when linking the 
-##                                 target
-##    SOURCES:STRING=<>          - List of source files
-##
-function(serialbox_add_library)

```

Comment:
there is no need to keep all this.... it is done in yoda:
https://github.com/cosunae/yoda/blob/create_library/cmake/yodaCreateLibrary.cmake